.gitignore

```diff
@@ -2,3 +2,8 @@ build/
 .clangd/
 .clang_complete
 /compile_commands.json
+*~
+*.pyc
+*.pyo
+build*/
+examples/grc/*.py
```

Comment:
When do these examples/grc/*.py get generated?

Reply:
These are generated on opening and running the flowgraph